github_helper.py

```diff
@@ -76,8 +70,7 @@ def post(self, content, endpoint):
             endpoint: URL to which to make the POST request. URL is relative
             to https://api.github.com/repos/{self.org}/{self.repo}/
         Returns:
-            True if request completed successfully.
-            False otherwise.
+            None if request returned successfully, error otherwise.
```

Comment:
Why not raise an exception here if it was unsuccessful? I think in Python using exceptions is generally more idiomatic. This looks more like a Go or C++ program using return codes.